recenter_tc/__init__.py

```diff
@@ -10,4 +10,47 @@
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-from ._version import __version__, __version_tuple__
```

Comment:
Do we not need the version imports here?

Reply:
Shoot, I missed that.  We do need them.  @AJThorpe did this fail flake8 because _version.py does not exist?

Reply:
I went ahead and put it after the docstring, as I may have been the one that accidentally deleted it when I added that docstring.

Reply:
./recenter_tc/__init__.py:58:1: F401 '._version.__version__' imported but unused
  ./recenter_tc/__init__.py:58:1: F401 '._version.__version_tuple__' imported but unused
  ./recenter_tc/compare_outputs_recenter_tc.py:[25](https://github.com/NRLMMD-GEOIPS/recenter_tc/actions/runs/6537613353/job/17751817532?pr=28#step:10:26):1: D[30](https://github.com/NRLMMD-GEOIPS/recenter_tc/actions/runs/6537613353/job/17751817532?pr=28#step:10:31)1 Use r""" if any backslashes in a docstring
  ./recenter_tc/compare_outputs_recenter_tc.py:35:38: W605 invalid escape sequence '\*'
  1     D301 Use r""" if any backslashes in a docstring
  2     F401 '._version.__version__' imported but unused
  1     W605 invalid escape sequence '\*'

Reply:
Needed to add the __all__ = ["__version__", "__version_tuple__"].  This works now